low-level/files/opt/seagate/sspl/bin/generate_resource_health_view/resource_health_view

```diff
@@ -911,6 +911,9 @@ class SSPLHealthView(object):
 
     def build_node_os_instances(self):
         os_drives_data = self.get_node_os_drive_data()
+        sspl_data_permission = f"chown -R sspl-ll:sspl-ll /var/{PRODUCT_FAMILY}/sspl/"
+        subout = self._run_command(sspl_data_permission)
+        del subout
```

Comment:
Didn't get purpose of assigning and deleting subout

Reply:
it is just deleting the extra created resource. to avoid memory leakage.

Reply:
just use `self._run_command(sspl_data_permission)` if output is is not needed

Reply:
@satish-seagate , i think what @sandeepsgt is saying is why do we need 3 lines of code... it can be done in 1 line without using the temp var?

Reply:
Done